### PR TITLE
feat: [PROD:8711] XLSX reader can now read Date type value and transfer it into date pattern

### PR DESCRIPTION
**Pull request by Guismus**
https://cosmo-tech.atlassian.net/browse/PROD-8711

### CODE REVIEW

**Review by csm-thu — COMMENTED** [review]
Suggested some changes.
Also, could you change the commit prefix from `feat` to `fix` please?

File: src/FileUtils/XLSXUtils.js [hunk 1]
```diff
@@ -25,6 +26,15 @@ const read = async (fileBlob, forceStr, emptyCols) => {
     data = data.map((row) => {
       return row.map((cell) => _castCellValueToStr(cell));
     });
+    cols.forEach((column, i) => {
+      if (column.type.includes('date'))
+        for (let j = 0 + hasHeader; j < data.length; j++)
+          if (data[j][i])
+            data[j][i] =
+              DateUtils.format(new Date(data[j][i]), options.dateFormat) !== ''
```
[the remark this answers is not in the document]
Comment (Guismus) [reply src/FileUtils/XLSXUtils.js]: Done

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Review by esasova — COMMENTED** [review]
I have a `console.error` when I upload a file with Date as cell formatting:
Screenshot from 2023-04-25 18-03-57

**Pull request merged**